PiwikTracker/Device.swift

```diff
@@ -1,9 +1,9 @@
 import UIKit
 
-internal struct Device {
+public struct Device {
```

Comment:
Not totally sure about these changes here. I like the idea that the user can select the slots for the custom dimensions themselves and can select the values they want to store.

At the same time I am not sure if I want to make Device and Application public.